packages/devtools/lib/src/timeline/event_details.dart

```diff
@@ -327,11 +369,33 @@ class _CpuFlameChart extends CoreElement {
       response,
       event.time.duration,
     );
+  }
 
+  void _drawFlameChart() {
     if (cpuProfileData.stackFrames.isEmpty) {
-      _updateChartWithError('CPU profile unavailable for time range'
-          ' [${event.time.start.inMicroseconds} - '
-          '${event.time.end.inMicroseconds}]');
+      final snapshotModeError = div()
+        ..add(span(
+            text:
+                'CPU profiling is not yet available for snapshots. You can only'
+                ' view '));
+      if (snapshot != null && snapshot.hasCpuProfile) {
+        snapshotModeError
+          ..add(span(text: 'the '))
+          ..add(span(text: 'CPU profile', c: 'error-action')
+            ..click(() => _loadCpuProfileSnapshotController.add(snapshot)))
+          ..add(span(text: ' included in the snapshot.'));
+      } else {
+        snapshotModeError.add(span(
+            text:
+                'a CPU profile if it is included in the imported snapshot file.'));
+      }
+
+      final generalError = div(
```

Comment:
this variable name adds no value. Maybe go ahead and put the alternate div in the conditional a couple lines bellow. That will also avoid creating the div when the message isn't needed as a small bonus.